### PR TITLE
Monorepo migration Cart/Table Component BENTO-2336 BENTO-2265

### PR DESCRIPTION
**Pull request by rana22**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

File: packages/bento-frontend/src/bento/dashboardTabData.js [hunk 1]
```diff
@@ -1206,7 +1206,7 @@ export const GET_FILE_IDS_FROM_FILE_NAME = gql`
   }`;
 
 // --------------- Tabs Table configuration --------------
-export const tabContainers = [
+export const tabContainersOld = [
```
Comment (iksheth) [inline packages/bento-frontend/src/bento/dashboardTabData.js:1209]: @rana22 Is this code used or we should remove it?

File: packages/bento-frontend/src/bento/fileCentricCartWorkflowData.js [hunk 2]
```diff
@@ -40,33 +53,54 @@ export const manifestData = {
   header: ['Study Code', 'Case ID', 'File Name', 'File ID', 'Md5sum', 'User Comments'],
 };
 
+// --------------- GraphQL query - Retrieve selected cases info --------------
```
Comment (iksheth) [inline packages/bento-frontend/src/bento/fileCentricCartWorkflowData.js:56]: Can we move query at the very bottom of the file, same like all the other custodian configurations follow?

File: packages/bento-frontend/src/components/Layout/LayoutView.js [hunk 3]
```diff
@@ -17,7 +17,7 @@ import DataDictonary from '../../pages/dataDictionary/dataDictonaryController';
 import Programs from '../../pages/programs/programsController';
 import ProgramDetail from '../../pages/programDetail/programDetailController';
 import GraphqlClient from '../GraphqlClient/GraphqlView';
-import fileCentricCart from '../../pages/fileCentricCart/cartController';
+// import fileCentricCart from '../../pages/fileCentricCart/cartController';
```
Comment (iksheth) [inline packages/bento-frontend/src/components/Layout/LayoutView.js:20]: Can we remove this line if not used?

File: packages/bento-frontend/src/pages/armDetail/FilesView/Wrapper.js
Comment (iksheth) [inline packages/bento-frontend/src/pages/armDetail/FilesView/Wrapper.js:1]: Can we rename this file as ThemeConfigWrapper?

File: packages/bento-frontend/src/pages/armDetail/armDetailView.js
Comment (iksheth) [inline packages/bento-frontend/src/pages/armDetail/armDetailView.js:1]: Can we remove un-used code that is commented out in this file?

File: packages/bento-frontend/src/pages/cart/cartView.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+        section="myFiles"
+      />
+    </>
+  );
+
+  /**
+  * configure table state
+  */
+  const initTblState = (initailState) => ({
+    ...initailState,
```
Comment (iksheth) [inline packages/bento-frontend/src/pages/cart/cartView.js:34]: @rana22 Correct me if I am wrong, But Shouldn’t we spread initailState prop at the very end of the object?

File: packages/bento-frontend/src/pages/caseDetail/FilesView/FilesTableView.js [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+import { themeConfig, wrapperThemConfig } from './Theme';
+import { TableView, TableContext } from '../../../bento-core/PaginationTable';
+import { footerConfig } from './Wrapper';
+
+const FilesTableView = ({
+  classes,
+  data,
+}) => {
+  const initTblState = (initailState) => ({
+    ...initailState,
```
Comment (iksheth) [inline packages/bento-frontend/src/pages/caseDetail/FilesView/FilesTableView.js:16]: @rana22 Correct me if I am wrong, But Shouldn’t we spread initailState prop at the very end of the object?

File: packages/bento-frontend/src/pages/caseDetail/caseDetailView.js [hunk 6]
```diff
@@ -71,7 +70,7 @@ const CaseDetail = ({ data, filesOfSamples, classes }) => {
 
   const samplesData = data.samples.map((s) => {
     const files = filesOfSamplesObj[s.sample_id];
-    const sample = _.cloneDeep(s);
+    const sample = { ...s };
```
Comment (iksheth) [inline packages/bento-frontend/src/pages/caseDetail/caseDetailView.js:73]: @rana22 why not directly assign "s" to sample?

File: packages/bento-frontend/src/store/index.js [hunk 7]
```diff
@@ -6,14 +6,17 @@ import layout from '../components/Layout/LayoutState';
 import dashboard from '../pages/dashboard/dashboardState';
 import stats from '../components/Stats/StatsState';
 import { sideBarReducerGenerator } from '../bento-core/FacetFilter/store/reducers/SideBarReducer';
+import { cartReducerGenerator } from '../bento-core/Cart/store/reducers';
 import { LocalFindReducerGenerator } from '@bento-core/local-find';
 
-const { statusReducer } = sideBarReducerGenerator();
 const { localFind } = LocalFindReducerGenerator();
+const { statusReducer } = sideBarReducerGenerator();
+const { cartReducer } = cartReducerGenerator();
```
Comment (iksheth) [inline packages/bento-frontend/src/store/index.js:14]: Aren’t we injecting cart reducers at run time?

**Review by iksheth — CHANGES_REQUESTED** [review]

File: packages/bento-frontend/src/bento/dashboardTabData.js (hunk 1 above)
Comment (iksheth) [reply packages/bento-frontend/src/bento/dashboardTabData.js:1209]: Used for Old Dashboard.

File: packages/bento-frontend/src/pages/cart/cartView.js (hunk 4 above)
Comment (rana22) [reply packages/bento-frontend/src/pages/cart/cartView.js:34]: It should be declare before like redux state { ...state, updateValues } so default values can be updated.

File: packages/bento-frontend/src/pages/caseDetail/caseDetailView.js (hunk 6 above)
Comment (rana22) [reply packages/bento-frontend/src/pages/caseDetail/caseDetailView.js]: updated

File: packages/bento-frontend/src/store/index.js (hunk 7 above)
Comment (rana22) [reply packages/bento-frontend/src/store/index.js:14]: @injectReducer method is used.

File: packages/bento-frontend/src/components/Layout/LayoutView.js (hunk 3 above)
Comment (rana22) [reply packages/bento-frontend/src/components/Layout/LayoutView.js]: removed old cart page

File: packages/bento-frontend/src/pages/armDetail/FilesView/Wrapper.js
Comment (rana22) [reply packages/bento-frontend/src/pages/armDetail/FilesView/Wrapper.js:1]: updated

**Review by iksheth — APPROVED** [review]

**Pull request merged**